src/main/java/formflow/library/config/FlowConfiguration.java

```diff
@@ -29,4 +32,13 @@ public class FlowConfiguration {
   public ScreenNavigationConfiguration getScreenNavigation(String screenName) {
     return flow.get(screenName);
   }
+
+  public void setFlow(Map<String, ScreenNavigationConfiguration> screenMap) {
```

Comment:
This method will get called by SnakeYAML when it builds the flow configuration from the yaml file. This allows us to inject information, if we want.